Tutorials/whatiswot/9-OSI_Layers/9-OSI-Layers.vtt

```diff
@@ -0,0 +1,161 @@
+WEBVTT
+
+00:00:00.000 --> 00:00:06.596 
+In WoT we are dealing with interacting with devices over the network.
+
+00:00:06.596 --> 00:00:13.193 
+Thus, we need to look at how such a communication is modelled.
+
+00:00:13.193 --> 00:00:21.306 
+The WoT focuses on the protocols that operate on the application layer that use TCP and UDP from
+
+00:00:21.306 --> 00:00:29.418 
+the transport layer. Thus, it is important to understand OSI layers and how sending and receiving data works.
+
+00:00:29.418 --> 00:00:35.743 
+Okay, so we were talking about sending JSON and changing the temperature before.
+
+00:00:35.743 --> 00:00:40.456 
+So let's see the magic behind the network.
+
+00:00:40.456 --> 00:00:45.899 
+Let's assume that we have the temperature data in the payload in the application layer of
+
+00:00:45.899 --> 00:00:51.343 
+the temperature sensor and we want to send it via a HTTP request to the dashboard.
```

Comment:
this is an error in the presentation as well. We want the dashboard to send the request, the temperature will reply with an HTTP response.